src/BinSkim.Driver/AnalyzeOptions.cs

```diff
@@ -48,5 +48,13 @@ public class AnalyzeOptions : AnalyzeOptionsBase
             "ignorePdbLoadError",
             HelpText = "If enabled, BinSkim won't break if we have a 'PdbLoadingException'.")]
         public bool IgnorePdbLoadError { get; set; }
+
+        [Option(
```

Comment:
>Option [](http://example.com/codeflow?start=9&length=6)

This is the main change of the PR.
The support of the v1 in Binskim, actually comes from SARIF SDK. Without remove the support in SDK we can not really remove it.
The change is to disable it in BinSkim. Please see more details in PR desc.
Now option `-v | --sarif-output-version` does not accept value `OneZeroZero` in BinSkim.

Reply:
I don't think you need this. We should delete it. 

Instead, just find the right place in our code and throw an InvalidOperationException in cases where someone passes SarifOneZeroZero on the command-line.

Reply:
changed accordingly.